ansible/roles/osdsdb/scenarios/etcd.yml

```diff
@@ -109,10 +109,3 @@
   when:
     - install_as_systemd == true and ( service_etcd_status.rc != 0 and etcd_service_source_exists.stat.exists is undefined or etcd_service_source_exists.stat.exists == false )  
   become: true
-
```

Comment:
Why is this removed?

Reply:
It was wrong, without Handler execution to restart the service, etcd health check will fail